app/server/lib/DocApi.ts

```diff
@@ -2932,6 +2933,6 @@ export async function downloadXLSX(activeDoc: ActiveDoc, req: Request,
                                    res: Response, options: DownloadOptions) {
   const {filename} = options;
   res.setHeader('Content-Type', 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet');
-  res.setHeader('Content-Disposition', contentDisposition(filename + '.xlsx'));
+  res.setHeader('Content-Disposition', filenameContentDisposition('attachment', filename + '.xlsx'));
```

Comment:
Can you confirm that `attachment` was the default `type` of `contentDisposition`?